neurom/features/morphology.py

```diff
@@ -380,11 +383,18 @@ def _mean_radius(points):
         valid_pts = np.ones(len(path_lengths), dtype=bool)
         if min_length_filter is not None:
             valid_pts = (valid_pts & (path_lengths >= min_length_filter))
+            if not valid_pts.any():
+                warnings.warn(
+                    "In 'trunk_origin_radii': the 'min_length_filter' value is greater than the "
+                    "path distance of the last point of the last section so the radius of this "
+                    "point is returned."
+                )
+                valid_pts[-1] = True
         if max_length_filter is not None:
             valid_pts = (valid_pts & (path_lengths <= max_length_filter))
         return points[valid_pts, COLS.R].mean()
 
-    return [_mean_radius(n.points)
+    return [_mean_radius(next(n.iter_sections()).points)
             for n in iter_neurites(morph, filt=is_type(neurite_type))]
```

Comment:
```python
return [mean_radius(n) for n in  iter_neurites(morph, filt=is_type(neurite_type))]
```

Reply:
Then make the `mean_radius` closure to take a neurite:

```python
def mean_radius(neurite):
    interval_lengths = morphmath.interval_lengths(neurite.root_node.points)
    ...
```
Make it explicit. This way will also make it easier to extend it for mixed subtrees (#981).

Reply:
Ah yeah good point, I forgot about `root_node`.